tools/ci_build/build.py

```diff
@@ -349,6 +349,9 @@ def parse_arguments():
     parser.add_argument(
         "--build_micro_benchmarks", action='store_true',
         help="Build ONNXRuntime micro-benchmarks.")
+    parser.add_argument(
+        "--reduce_ops", action='store_true',
+        help="Exclude unused ops.")
     parser.add_argument(
```

Comment:
--exclude_unused_ops would be clearer. Would also be good to say it's implied if you provide the --include_ops_* params, and for the help so say you can run the standalone script separately and just use this option.

Reply:
Actually why do we need this flag at all? 

If you ran the script standalone the ops are already excluded. If you provided the `--include_ops_*`  flags we do that as part of the build. What value is this flag adding?

---
In reply to: [476933759](https://github.com/microsoft/onnxruntime/pull/4917#discussion_r476933759) [](ancestors = 476933759)

Reply:
The flag getting reused in "run_onnxruntime_tests(...)" to skip python-binding related tests.
Do not want to do "if include... or include ..." more than once.

Reply:
Removed.

Reply:
Still seeing the flag. Was it meant to be removed?

Reply:
Sorry, now it's gone.